plugins/wallabag.koplugin/main.lua

```diff
@@ -398,36 +421,41 @@ function Wallabag:synchronise()
 
     local remote_article_ids = {}
     local downloaded_count = 0
+    local failed_count = 0
     if self.access_token ~= "" then
         local articles = self:getArticleList()
-        logger.dbg("number of articles: ", articles.total)
-        --logger.dbg("articles: ", articles)
-
-        info = InfoMessage:new{ text = _("Downloading articles…") }
-        UIManager:show(info)
-        UIManager:forceRePaint()
-        UIManager:close(info)
-        for _, article in ipairs(articles._embedded.items) do
-            logger.dbg("article: ", article.id)
-            remote_article_ids[ tostring( article.id ) ] = true
-            if self:download(article) then
-                downloaded_count = downloaded_count + 1
+        if articles then
+            logger.dbg("number of articles: ", articles.total)
```

Comment:
For clarity and grep it'd probably be a good idea to prefix all or most of these with wallabag.